deploy.py

```diff
@@ -49,17 +50,22 @@
 print("Done")
 
 print("Making file system writable ... ", end="")
-ssh.exec_command("rw")
+ssh.exec_command("sudo mount -o remount,rw / ; sudo mount -o remount,rw /boot")
```

Comment:
I'm not convinced this actually works, and you're not waiting for this to finish either.